### DISCUSSION TITLE
NetAlertx and PI-Hole on Raspberry PI

### DISCUSSION THREAD

**Discussion by itpilot** [netalertx/NetAlertX#895] [Q&A]
guys please can you help the noob

firstly, Wow!! NetAlertX is a Great Product
Thank You

i am using NetAlertX on a Raspberry PI 4

i have three questions

1. Can i run PI-Hole on the Same Raspberry PI 4 without any Issues?
2. When i receive an Alert from NetAlertX of a New Device(s), is there an Option to Block ALL New Devices from having Internet Service?
IE: The New Device(s) will receive an IP Address however will be in a Quarantine State Until Approved Internet Service(s) are given
3. Can i create GROUPS and Place my Devices eg: Mobiles, Printers, Laptops etc. in the Groups?

any help greatly appreciated

**Comment by jokob-sk** [comment]
Hi there,

1. Yes
2. Theoretically yes, but NetAlertX can't block devices. So you could
integrate it with e. g. Home Assistant or n8n to create a bridge to your
router that would then block the devices.
3. Yes

**Comment by itpilot** [comment]
@jokob-sk thank you for your reply, much appreciated

Your answer - So you could integrate it with e. g. Home Assistant or n8n to create a bridge to your router that would then block the devices.

Is it not Possible to use **PI-HOLE** for my requirements?

or is the Better option to use  **Home Assistant?**

any help greatly appreciated

**Comment by jokob-sk** [comment]
You can use PiHole with netalertx, but I don't think PiHole supports
blocking clients from the network.
